test/v4/controllers/ListBsasControllerSpec.scala

```diff
@@ -143,7 +144,7 @@ class ListBsasControllerSpec
       incomeSourceType = Some(typeOfBusiness)
     )
 
-    val response: ListBsasResponse[BsasSummary] = ListBsasResponse(
+    val response: ListBsasResponse[BsasSummary] = models.response.listBsas.ListBsasResponse(
```

Comment:
any reason why this has an explicit call when ListBsasResponse is directly imported above?